api/.env.local

```diff
@@ -4,6 +4,7 @@ MINTER_ACCOUNT_KEY_INDEX=0
 
 FUNGIBLE_TOKEN_ADDRESS=${FUNGIBLE_TOKEN_ADDRESS}
 NON_FUNGIBLE_TOKEN_ADDRESS=${NON_FUNGIBLE_TOKEN_ADDRESS}
+FUSD_TOKEN_ADDRESS=${FUSD_ADDRESS}
```

Comment:
I noticed that `FUSD_ADDRESS` in many other places -- does this need to be `FUSD_TOKEN_ADDRESS`?

Reply:
No this is fine. Prevents infinite loop when expanding env vars.